docs/starters.yml

```
@@ -2222,3 +2222,20 @@
     - Configuration for Firebase hosting
     - Configuration for Cloud Build deployment
     - Clear documentation to have your site deployed on Firebase behind SSL in no time!
+- url: https://lewis-gatsby-starter-blog.firebaseapp.com
+  repo: https://github.com/lewislbr/lewis-gatsby-starter-blog
+  description: A simple custom Gatsby starter template to start a new blog or personal website.
+  tags:
+    - Blog
+    - Styling:CSS-in-JS
+    - Markdown
+    - PWA
+    - Portfolio
+    - Seo
```

```suggestion
    - SEO
```
to keep casing consistent and don't create extra tag